src/main/java/com/codurance/retropolis/controllers/CardController.java

```diff
@@ -43,6 +45,13 @@ public Card postCard(@RequestBody @Valid NewCardRequestObject request) {
     return new ResponseEntity<>(HttpStatus.OK);
   }
 
+  // todo add validation
+  @PatchMapping(value = "/{cardId}")
+  public Card updateCard(@PathVariable Long cardId, @RequestBody UpdateCardRequestObject request) {
+    return cardService.update(cardId, request);
+  }
```

Comment:
We may want to change the name of this endpoint. To be precise this method is updating the text on the card. Also we could be more specific about these parameters (perhaps with @RequestParam?